sktime/transformations/hierarchical/aggregate.py

```diff
@@ -41,8 +42,8 @@ class Aggregator(BaseTransformer):
             "pd_multiindex_hier",
         ],
         "y_inner_mtype": "None",  # which mtypes do _fit/_predict support for y?
-        "capability:inverse_transform": False,  # does transformer have inverse
-        "skip-inverse-transform": True,  # is inverse-transform skipped when called?
+        "capability:inverse_transform": True,  # does transformer have inverse
+        "skip-inverse-transform": False,  # is inverse-transform skipped when called?
```

Comment:
I would leave the `skip-inverse-transform` as `True`.

Why: otherwise a `TransformedTargetPipeline` with the aggregator at the start will no longer produce forecasts of totals.